packages/management/tkg-clusterclass-aws/metadata.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: data.packaging.carvel.dev/v1alpha1
+kind: PackageMetadata
+metadata:
+  name: tkg-clusterclass-aws.tanzu.vmware.com
+  namespace: tkg-clusterclass-aws
+spec:
+  displayName: "tkg-clusterclass-aws"
+  longDescription: "Clusterclass for Azure provider package defines the default clusterclass resource"
```

Comment:
Mostly just spellng consistency stuff like this. I can see we plan to refresh this once the actual payload is incorporated, so your call on when to update these.

Clusterclass -> ClusterClass
defines the default set of Azure ClusterClass resources for Tanzu